src/app/(sidebar)/account/styles.scss

```
@@ -0,0 +1,13 @@
+@use "../../../styles/utils.scss" as *;
+
+.Account {
+  &__Card {
```

In BEM naming convention we use camel case for nested block elements.
```suggestion
  &__card {
```